app/models/katello/content_view_environment.rb

```diff
@@ -26,11 +29,16 @@ class ContentViewEnvironment < Katello::Model
     scope :non_default, -> { joins(:content_view).where("katello_content_views.default" => false) }
     scope :default, -> { joins(:content_view).where("katello_content_views.default" => true) }
     alias :lifecycle_environment :environment
+    delegate :organization, :to => :environment
 
     def self.for_content_facets(content_facets)
       joins(:content_view_environment_content_facets, :content_facets).where("#{Katello::ContentViewEnvironmentContentFacet.table_name}.content_facet_id" => content_facets).uniq
     end
 
+    def self.for_activation_keys(activation_keys)
```

Comment:
It seems this method isn't used anywhere. (also, it ignores ordering) I'd say remove it.

Reply:
removed